core/src/consensus/tendermint/worker.rs

```diff
@@ -1005,16 +1007,16 @@ impl Worker {
             self.move_to_height(height);
             let prev_block_view = TendermintSealView::new(proposal.seal()).previous_block_view().unwrap();
             self.save_last_confirmed_view(prev_block_view);
-            let proposal_at_view_0 = self
+            let is_view0_proposal = self
```

Comment:
```suggestion
            let view0_proposal_exists = self
```

Reply:
Same as the upper comment. `view0_proposal_exists` seems that there is another proposal for view0 other than the `proposal` parameter. How about using `view0_proposal` as the name.